core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTestBase.scala

```diff
@@ -150,7 +150,7 @@ trait SplitClusterDUnitTestBase extends Logging {
     doTestRowTableCreation()
   }
 
-  def _testComplexTypesForColumnTables_SNAP643(): Unit = {
+  def testComplexTypesForColumnTables_SNAP643(): Unit = {
```

Comment:
Yes, the first two no-op are covered in the security dunit test.
Eventually, this test will be modified so that even the parent testcase should run fine with it but the suggestion was to make it no-op as it was disabled until now and it could result in an unnecessary failure purely because of lack credentials supplied.